packages/react-draggable-dialog/src/components/DraggableDialog/DraggableDialog.test.tsx

```diff
@@ -1,16 +1,183 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { act, render, renderHook } from '@testing-library/react';
+
 import { DraggableDialog } from './DraggableDialog';
+import { useDraggableDialog } from './useDraggableDialog';
+import { DragEndEvent } from '@dnd-kit/core';
+
+jest.mock('@fluentui/react-components', () => ({
```

Comment:
is this mock necessary (mocks most of the time show/expose not the best API/red flags) ? cant we use use`expect.any<String>` in order to test those dictionary outputs of the hook ?

Reply:
I'm not sure if that is necessary and if we can have an alternative approach. 
The gymnastics here is to make sure that the `useId` hook always returns the same value for the same component and that the value is predictable. This way the tests are both not flaky and the ID returned stays the same, even if useId changes its implementation internally.
Is there any alternative way to achieve the same in this case?

Reply:
yes I mentioned it in my comment. 

regarding flaky - testing internals of useId is probably flaky on itself/codesmell.

`cant we use useexpect.any<String> in order to test those dictionary outputs of the hook ?`

https://jestjs.io/docs/expect#expectanyconstructor